tests/unit/cli/cli-test.js

```diff
@@ -133,6 +145,29 @@ describe('Unit: CLI', function() {
     td.verify(init(), {ignoreExtraArgs: true, times: 0});
   });
 
+  describe('process signal listeners', function() {
+    var onCommandInterrupt;
+    beforeEach(function() {
+      onCommandInterrupt = td.matchers.isA(Function);
+    });
+
+    it('sets up signal listeners before command started', function() {
+      stubValidateAndRun('serve', function() {
```

Comment:
Yeah `instrumentation` isn't a notifier.  But there is actually a `beforeRun` method that's invoked before a command is run.  see https://github.com/ember-cli/ember-cli/blob/master/tests/unit/cli/cli-test.js#L184-L221